language-support/ts/codegen/src/Main.hs

```diff
@@ -92,8 +92,14 @@ genModule mod
             ["// Generated from " <> T.intercalate "/" (unModuleName curModName) <> ".daml"
             ,"/* eslint-disable @typescript-eslint/camelcase */"
             ,"/* eslint-disable @typescript-eslint/no-use-before-define */"
-            ,"import * as daml from '@digitalasset/daml-json-types';"
+            ,"/* eslint-disable @typescript-eslint/no-namespace */"
```

Comment:
I'd prefer to add these exceptions in a very targeted way to the lines they apply to.

Reply:
Is it possible to add those exceptions right in front of the namespace declaration?